File: striver/S3L1/move_zeros_to_end.java
package S3L1;

public class move_zeros_to_end{
    public void moveZeroes(int[] nums) {
        int sr=0;
        for(int fr = 0;fr<nums.length;fr++){
            if(nums[fr]!=0){
                nums[sr++]=nums[fr];
            }
        }
        while(sr<nums.length){
            nums[sr++] = 0;
        }
    }
}

File: striver/S3L1/move_zeros_to_end.py
class Solution(object):
    def moveZeroes(self, nums):
        sr=0
        for fr in range(0,len(nums)):
            if(nums[fr]!=0):
                nums[sr]=nums[fr]
                sr+=1
        while sr<len(nums):
            nums[sr]=0
            sr+=1